event.go

```diff
@@ -0,0 +1,52 @@
+package basictracer
+
+import "github.com/opentracing/opentracing-go"
+
+// An Event is emitted when a mutating command is called on a Span.
+type Event interface{}
+
+// EventCreate is emitted when a Span is created.
+type EventCreate struct{ OperationName string }
```

Comment:
Since this is for when a span is created, I think that `EventCreate` should either hold or be similar to `opentracing.StartSpanOptions`. For example, in the `NetTraceIntegrator`, it would be useful to be able to override the start time of the `trace.Trace` a la `tr.Start = t.StartTime`.